kubernetes-templates/README.md

```diff
@@ -0,0 +1,40 @@
+# Deploying cvat in a kubernetes cluster
+
+This guide will focus on how to deploy cvat in an kubernetes environment.
+
+## Building the container
+In order to do so, you will first need to build the cvat backend and frontend images and push them to a registry that you can pull from within the cluster
+```
+echo "Building backend"
+docker build --cache-from $CI_REGISTRY_IMAGE/backend:release-1.1.0 \
+  --build-arg TF_ANNOTATION=no --build-arg AUTO_SEGMENTATION=no \
+  --build-arg WITH_TESTS=no --build-arg TZ="Etc/UTC" --build-arg OPENVINO_TOOLKIT=no \
+  --build-arg USER=django --build-arg DJANGO_CONFIGURATION=production \
+  --build-arg TZ="Etc/UTC" .
+docker push $CI_REGISTRY_IMAGE/backend:release-1.1.0
+
+echo "Building frontend"
+docker build --file Dockerfile.ui \
+  --tag $CI_REGISTRY_IMAGE/frontend:release-1.1.0 - .
```

Comment:
I get an error "docker build requires exactly 1 argument." I guess this should read
`docker build --file Dockerfile.ui \
  --tag $CI_REGISTRY_IMAGE/frontend:release-1.1.0 .` (without the dash) which succeeds in building for me.

Reply:
Thanks for pointing it out, i must have forgotten to remove the dash. I am used to adding the cache from option to every build, but did not want to commit it, to keep the template simple :) `--cache-from $CI_REGISTRY_IMAGE/frontend:release-1.1.0`

Reply:
Has been fixed in d1a855cf24a06f16812aaa3fd58530533116f8ed